src/screens/DetailedViewScreen.tsx

```diff
@@ -1,75 +1,84 @@
 //This is where we will be displaying the information of each single place
-import React, { useState, useEffect, useMemo } from "react";
-import { StyleSheet, Dimensions, View, Text, ActivityIndicator } from "react-native";
-import axios from "axios";
+import React from "react";
+import { StyleSheet, Dimensions, View, Text, ActivityIndicator, TextProps } from "react-native";
 import { computeDistanceMi } from "../util/distance";
-import { SERVER_BASE_URL } from "../util/env";
 import { useLocation } from "../hooks/useLocation";
-import { PlaceDetailsResponseData } from "@googlemaps/google-maps-services-js";
 import { Button } from "../components/Button";
 import { TEXT_COLOR } from "../util/colors";
+import { PlaceImage } from "../components/PlaceImage";
+import { useFetchPlace } from "../hooks/useFetchPlace";
+import { getAverageA11yRating } from "../util/processA11yRatings";
 
 export interface PlaceProps {
-	placeID?: string;
-	setPage: () => void;
+	placeID: string;
+	setPage: (newPage: string) => void;
 }
 
+// eslint-disable-next-line
```

Comment:
could you add the specific rule that we are violating here so that we don't ignore all rules?

Reply:
actually was able to remove this, it was a remnant of an intermediate change i removed in the end